microscopium/preprocess.py

```diff
@@ -889,6 +889,76 @@ def create_missing_mask(missing, order, rows=512, cols=512):
     return mask
 
 
+def montage_with_missing(fns, order=None):
+    """Montage a list of images, replacing missing fields with dummy images.
+
+    The methods `montage` and `montage_stream` assume that image filenames
+    and image iterators passed to it are complete, and include the full set
+    images belonging to the well. Some screens have missing fields,
+    so this function can be used to montage together images with missing
+    fields. Missing fields are determined from the information in the image
+    file name. See 'find_missing_fields'
+
+    Parameters
+    ----------
+    fns : list of str
+        The list of filenames to montage.
+    order : array-like of int, shape (M, N), optional
+        The order of the stitching, with each entry referring
+        to the index of file in the fns array.
+
+    Returns
+    -------
+    montaged : array-like, shape (P, Q)
+        The montaged image.
+    mask : array of bool
```

Comment:
No you were right the first time, the shape matches the montaged image! I'll add the shape to the docstring.